src/modules/site-v2/base/views/tools/genetic_mapping.py

```diff
@@ -76,7 +76,9 @@ def genetic_mapping():
     ],
 
     # Sample data
-    'sample_data_url': generate_blob_url(MODULE_SITE_BUCKET_ASSETS_NAME, 'data/nemascan_sample_data.tsv'),
+    'elegans_sample_data_url': generate_blob_url(MODULE_SITE_BUCKET_ASSETS_NAME, 'data/nemascan_sample_data_elegans.tsv'),

```

Comment:
we will need to find a way to ensure that this file in GCP is always in-sync with the [newer] ones from Github.